blueprints/terminal/gomplate/ops2deb.yml

```diff
@@ -0,0 +1,18 @@
+name: gomplate
+matrix:
+  versions:
+    - 3.11.5
```

Comment:
Could you order version from oldest to newest? Auto update is confused when it's not the case.

Reply:
What about my previous merges? Should I fix them also

Reply:
No worries, I did it :)